### DISCUSSION TITLE
Can I contact an organization owner that is using my company name?

### DISCUSSION THREAD

**Discussion by 0x80** [community/community#140135] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I see that someone set up an organization with my company name. It's an empty account without any repositories and 0 contributions in the last year. So it seems to be just a placeholder, possibly put there by the person who sold me one of the domains...

I would like to start using that name on github, but I don't now how to get in contact with the user. 

I also already own the NPM namespace, and have published packages under that namespace.

**Comment by feijoes** [comment]
You could try [reporting](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam) the organization, explaining the situation, and providing proof of ownership (such as screenshots of your NPM organization or domain ownership). But aside from that, I'm not sure what else you can do

**Reply by 0x80** [reply]
Thanks that's something I was considering, but didn't see a category of abuse that was even remotely fitting. I'll give that a try anyway.

**Comment by ebndev** [comment, marked as answer]
Hi @0x80,

We are no longer accepting requests to release dormant usernames. We are not able to accommodate this request.

If you hold a trademark to this name and believe there is a clear case of confusion between the account and your trademark, please review our [Trademark Policy](https://docs.github.com/articles/github-trademark-policy/) to determine if you should submit a trademark policy violation report.

Thank you!

**Reply by 0x80** [reply]
@ebndev Thanks for letting me know!

**Reply by dainiusfigoras** [reply]
I am in similar situation and wander if there is a way to contact owner of that org in github, so maybe he will be willing to give or free name?

**Reply by 0x80** [reply]
Yeah I'm not sure why forwarding a simple message to the owner is not on the menu. I do not need github to do anything for me except forward it with my details. 

I suspect that in a majority of cases, someone just made the account when they had different plans, and then abandoned it. It seems a bit silly to let them hold on to these organization names indefinitely if they are not using them and are also not made aware that other people would like to use them.

@ebndev I hope you can consider providing a service where you send a simple message to the owner saying "this person would like to contact you about adopting your github organization name"
